boot/boot_test.go

```diff
@@ -86,6 +86,22 @@ func (s *baseBootenvSuite) mockCmdline(c *C, cmdline string) {
 	c.Assert(ioutil.WriteFile(s.cmdlineFile, []byte(cmdline), 0644), IsNil)
 }
 
+// mockAssetsCache mock boot assets in cache, a list of (path, data) tuples,
+// where if data is omitted an empty files gets created.
+func mockAssetsCache(c *C, rootdir, bootloaderName string, cachedAssets [][]string) {
+	p := filepath.Join(dirs.SnapBootAssetsDirUnder(rootdir), bootloaderName)
+	err := os.MkdirAll(p, 0755)
+	c.Assert(err, IsNil)
+	for _, cachedAsset := range cachedAssets {
+		var content []byte
+		if len(cachedAsset) == 2 {
```

Comment:
Why 2 and not e.g. 'if len(cachedAsset) > 0'? Can you add a comment in the code?